lib/ex_doc/utils/code.ex

```diff
@@ -3,6 +3,12 @@ defmodule ExDoc.Utils.Code do
 
   # TODO: this is vendored from Elixir v1.11.0.
   #       Remove and use Code.fetch_docs/1 in the future.
+
+  # special case required for Elixir
+  def fetch_docs(Elixir) do
+    false
+  end
```

Comment:
we don't need to special case it in two places, the other place is enough, but I'm planning to send this particular change upstream (this fetch_docs/1 is vendored) so we may as well keep them in sync.